README.md

```diff
@@ -1,4 +1,58 @@
-# devtools-protocol
+# DevTools Protocol
 
-This is the repository for devtools-protocol. You're welcome to
-[[contribute]](CONTRIBUTING.md)!
+This repository is for resources and discussion around aligning the developer tools protocol used by various browsers, JavaScript engines and debugging tools.
+
+## Goals
+
+- Help empower web developers to build great applications by supporting an ecosystem of diagnostics tools which work well across a variety of targets.
+- Reduce the need for target-specific "devtools protocol adapters" in order to simplify the experience for tool users and tool developers.
+- Provide a vendor-neutral forum to facilitate collaboration and consensus building between different protocol implementations and interested clients.
+- Reduce the engineering investment necessary to build compatible protocol implementations through sharing of test cases.
+
+[Contributions](CONTRIBUTING.md) from the community are welcome!
+
+## What is here in this repo
+
+Not much yet, just a [few issues](https://github.com/WICG/devtools-protocol/issues) tracking work we'd like to do and links to a few resources below.
+
+## Implementations
+Engines and browsers that all have a similar debugging protocol built-in:
+- [Chrome DevTools](https://chromedevtools.github.io/devtools-protocol/), also used by most other chromium-based browsers
+- [WebKit / Safari](https://github.com/WebKit/webkit/tree/master/Source/JavaScriptCore/inspector/protocol)
+- [Node.js](https://chromedevtools.github.io/devtools-protocol/v8/)
+- Firefox - [in development](https://groups.google.com/forum/#!msg/mozilla.dev.platform/4-4A8W-nP5g/Y9C9UkWTAAAJ)
+- Edge - in development (link forthcoming)
+
+Adapters that expose a common protocol:
+- [Microsoft Edge Diagnostics Adapter](https://github.com/Microsoft/edge-diagnostics-adapter) - to be replaced with native support (above).
+- [RemoteDebug iOS WebKit Adapter](https://github.com/RemoteDebug/remotedebug-ios-webkit-adapter)
+- See also [RemoteDebug list of adapters](https://remotedebug.org/adaptors/)
+
+See also [RemoteDebug Protocol Compatibility Tables](http://compatibility.remotedebug.org/)
```

Comment:
This link should really be https://remotedebug.org/integrations/, as the compat tables are a API comparison tool.

Reply:
That link is already below.  I moved this link up just for context for "similar".